### DISCUSSION TITLE
can't touch rustdesk settings connected by remote

### DISCUSSION THREAD

**Discussion by Joselito1980** [rustdesk/rustdesk#6427] [General]
### Bug Description

I try to explain as best as possible, my English is bad :-)

When I connect via remote access with trustdesk and try to access the configuration of the computer that I am connected to, when I move the mouse over it, it goes dark and I cannot touch anything (or stop the service, restart it, change options...) if I move the mouse on the equipment physically works well.

### How to Reproduce

Connect remotely to a computer and try to modify Rustdesk settings on the connected computer.

### Expected Behavior

being able to touch the program settings without problems

### Operating system(s) on local side and remote side

Windows 10 Pro and Windows 11 pro

### RustDesk Version(s) on local side and remote side

rustdesk-1.2.3 and rustdesk-1.2.4

### Screenshots

![image](https://github.com/rustdesk/rustdesk/assets/60007170/d2887a45-0808-4b26-842e-0114b2e753f7)

### Additional Context

I have connected to the same computer by teamviewer and I can touch the settings. It only happens when I'm logged in via rustdesk

**Comment by Joselito1980** [comment]
sorry... its a option in the configuration..... :-)

**Reply by weidinger-c** [reply]
Which one is it?
